gateway/gateway.proto

```diff
@@ -131,6 +131,9 @@ message EvaluateRequest {
     string channel_id = 2;
     // The signed proposal ready for evaluation.
     protos.SignedProposal proposed_transaction = 3;
+    // If targeting the peers of specific organizations (e.g. for private data scenarios),
+    // the list of organizations should be supplied here.
```

Comment:
Can we expand the comment to mention the MSPID is passed to indicate the org?  Same comment on the proposal side.